README.md

```diff
@@ -72,11 +72,13 @@ For more details on how to use `genezio`, you can check out the official [docume
 
 # Features
 
+-   🚀 **Function-as-a-service**: No need to worry about infrastructure, scaling, or maintenance.
+-   📦 **Genezio Functions**: Deploy and run code on-demand without managing servers or infrastructure.
 -   💡 Full static type-safety with **auto-completion** in your favorite editor.
 -   🧩 **Typesafe RPC**: Ensuring type safety and IDE auto-completion across diverse languages like TypeScript, Dart, Kotlin and Go, by leveraging advanced code analysis.
--   🚀 Tested and production ready for Typescript.
+-   🚀 Tested and production ready for Javascript/Typescript.
     -   Beta support for: GoLang, Kotlin and Dart.
--   📦 Export the resulting SDK using dependency managers such as **NPM Registry** - private to your team or public for anybody to use.
+-   📦 **NPM Registry** - the client SDK is pushed to a private to your team or public for anybody to use.
```

Comment:
`to a private` what?

Reply:
Today I am eating words, not food. Thanks a lot :D